tests/python/gaia-ui-tests/gaiatest/apps/music/app.py

```diff
@@ -77,3 +78,19 @@ def tap_artists_tab(self):
         Wait(self.marionette).until(expected.element_displayed(element))
         element.tap()
         return ListView(self.marionette)
+
+    def a11y_click_albums_tab(self):
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._albums_tab_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
+        self.accessibility.click(element)
+        return ListView(self.marionette)
+
+    def a11y_click_first_album(self):
```

Comment:
Could you move this into regions/list_view.py and you won't need to import a SublistView